elections/README.md

```diff
@@ -0,0 +1,80 @@
+* [Kata Containers Elections](#kata-containers-elections)
+  * [Architecture Committee Elections](#architecture-committee-elections)
+      * [Election Dates](#election-dates)
+      * [Electorate](#electorate)
+      * [Candidacy](#candidacy)
+      * [Architecture Committee Responsibilities](#architecture-committee-responsibilities)
+      * [System](#system)
+
+
+
+# Kata Containers Elections
+
+This repository contains Kata Containers Elections reference documents
+and tooling to run elections.
+
+
+# Architecture Committee Elections
+
+
+## Election Dates
+
+The Architecture Committee elections take place every 6 months, with 3 seats
+up for election in September and 2 seats up for election in February.
```

Comment:
This duplicates information in https://github.com/kata-containers/community#architecture-committee. Could you define it in one location only and reference it from other docs to minimise the maintenance cost?

Reply:
ack